apps/site/pages/en/learn/migrations/introduction.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Userland Migrations
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Userland Migrations
+
+Node.js provides migrations for "userland" (what you write vs node's own) source-code to facilitate adoption of new features and upgrading source-code affected by breaking changes. These are done in collaboration with [`codemod`](https://www.codemod.com), who also work with other major projects like Next.js, React, and Tailwind. Node.js's migrations live in the [`nodejs/userland-migrations`](https://github.com/nodejs/userland-migrations) repository and are overseen by the [`@nodejs/userland-migrations`](https://github.com/orgs/nodejs/teams/userland-migrations) team.
```

Comment:
user land team is private it's will be a 404 if not in node gh org

Reply:
It looks like all teams within the nodejs org are private. Maybe it's not terribly important that it 404s for non-members? At least for members it can still have utility.